CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# dbt_shopify_holistic_reporting v0.9.0
+[PR #33](https://github.com/fivetran/dbt_shopify_holistic_reporting/pull/33) includes the following updates:
+
+## dbt Fusion Compatibility Updates
+- Updated package to maintain compatibility with dbt-core versions both before and after v1.10.6, which introduced a breaking change to multi-argument test syntax (e.g., `unique_combination_of_columns`).
+- Temporarily removed unsupported tests to avoid errors and ensure smoother upgrades across different dbt-core versions. These tests will be reintroduced once a safe migration path is available.
+  - Removed all `dbt_utils.unique_combination_of_columns` tests.
+
+## Upstream Dependency Changes
+- Increased the `dbt_klaviyo` dependency from `[">=0.9.0", "<0.10.0"]` -> `[">=1.0.0", "<1.1.0"]`. Refer to the below Klaviyo releases to understand the full scope of changes.
+  - [dbt_klaviyo v1.0.0](https://github.com/fivetran/dbt_klaviyo/releases/tag/v1.0.0)
+- Increased the `dbt_shopify` dependency from `[">=0.11.0", "<0.14.0"]` -> `[">=0.21.0", "<0.22.0"]`. Refer to the below Shopify releases to understand the full scope of changes.
```

Comment:
Should we add the release links back to v0.11.0?

Reply:
I opted for only 15 since the previous range went to <14. However, I realize I didn't have 14 included below. I will add that in.

Reply:
Added 14

Reply:
Got it makes sense!